src/integrations/feed/generators.ts

```diff
@@ -151,5 +157,17 @@ export const feedGenerators: Record<FeedVersion, FeedGenerator> = Object.freeze(
 );
 
 export const versionToFeedGenerator = (version: number): FeedGenerator => {
+  if (version >= 30) {
+    return new FeedGenerator(
+      feedClient,
+      new FeedLofnConfigGenerator(
+        {},
+        {
+          feed_version: version.toString() as FeedVersion,
+        },
+      ),
+    );
```

Comment:
During the cleanup, I would rethink it a little bit

Reply:
Yes, basically the issue was that FeedConfig and feed_config_name gets passed around for other generators and I did not want to add `feed_version` to FeedConfig type. Will note it.